### PR TITLE
satisfy pessimistic dependency warnings on build

### PR DESCRIPTION
**Pull request by scosist**
## Pull Request Checklist

**Is this in reference to an existing issue?**

#### General

- [ ] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)

- [ ] Update README with any necessary configuration snippets

- [ ] Binstubs are created if needed

- [ ] RuboCop passes

- [ ] Existing tests pass

#### New Plugins

- [ ] Tests

- [ ] Add the plugin to the README

- [ ] Does it have a complete header as outlined [here](http://sensu-plugins.io/docs/developer_guidelines.html#coding-style)

### CODE REVIEW

**Comment by scosist** [comment]
I did some testing but couldn't build without these changes. I temporarily commented out lines in the gemspec related to key signing in order to build and test.

Do I need to update the README regarding these dependency changes?

File: sensu-plugins-nrpe.gemspec [hunk 1]
```diff
@@ -39,7 +39,7 @@ Gem::Specification.new do |s|
   s.test_files             = s.files.grep(%r{^(test|spec|features)/})
   s.version                = SensuPluginsNrpe::Version::VER_STRING
 
-  s.add_runtime_dependency 'sensu-plugin',      '~> 1.2.0'
+  s.add_runtime_dependency 'sensu-plugin',      '~> 1.2', '>= 1.2.0'
```
Comment (majormoses) [inline sensu-plugins-nrpe.gemspec]: You can actually just remove the last 0 and that will mean `>= 1.2, < 2.0`

File: sensu-plugins-nrpe.gemspec [hunk 2]
```diff
@@ -48,7 +48,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'pry',                       '~> 0.10'
   s.add_development_dependency 'rake',                      '~> 10.0'
   s.add_development_dependency 'redcarpet',                 '~> 3.2'
-  s.add_development_dependency 'rubocop',                   '~> 0.40.0'
+  s.add_development_dependency 'rubocop',                   '~> 0.40', '>= 0.40.0'
```
Comment (majormoses) [inline sensu-plugins-nrpe.gemspec]: same thing.

**Comment by majormoses** [comment]
@scosist appreciated, ya I need to get another maintainer (who has more access than I do) and has been through setting these up before to work out the travis/gem issues.

**Comment by majormoses** [comment]
If you want I can just incorp those changes in with my pr if you want.

**Comment by scosist** [comment]
Either way, whichever is easiest.

**Comment by majormoses** [comment]
I can rebase my branch off master. until we get the travis issues we can merge but no releases will be made

**Pull request merged**